### ISSUE TITLE
[Depends on #915] FAQ page links go to homepage

### ISSUE THREAD

**Issue by Zeyber** [SAP/cloud-commerce-spartacus-storefront#3482]
Go to FAQ and click any link. You get redirected back to the homepage.

**Comment by mszczepaniak** [comment]
I cannot reproduce it right now, since we get not found after clicking on the FAQ link. It probably needs a backend change.
![2019-07-12 09_33_36-Not Found](https://user-images.githubusercontent.com/4272069/61111122-20d7d000-a489-11e9-84b1-b73f832b5132.png)

**Comment by Platonn** [comment]
@mszczepaniak on dev19 and modelt the page label for FAQ is correct (the same as default semantic link configured in Spartacus) - `/faq`. However in dev17 the page label is incorrect `/faq/corporate` (don't know why? FYI @hackergil @giancorderoortiz ). @mszczepaniak can you verify that you were using dev17 and got not found page?

**Comment by Platonn** [comment]
Duplicate of https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/915

**Comment by mszczepaniak** [comment]
@Platonn yes, I was using dev17.

**Comment by giancorderoortiz** [comment]
I'm redirected to faq page on dev-com19
![Screen Shot 2019-07-12 at 2 58 40 PM](https://user-images.githubusercontent.com/46171897/61152175-b6895480-a4b5-11e9-91b8-0fdce33cd2b5.png)

**Comment by hackergil** [comment]
This will be addressed by #915

**Comment by Xymmer** [comment]
dupe

**Comment by giancorderoortiz** [comment]
Still an issue?
FAQ not in checkout flow, hence a minor.

**Comment by giancorderoortiz** [comment]
I still see errors in FAQ page. None of the links in that page work. 

![Screen Shot 2021-06-06 at 9 34 39 AM](https://user-images.githubusercontent.com/46171897/120926361-80a57880-c6aa-11eb-8e8f-ee03bc1482e2.png)
![Screen Shot 2021-06-06 at 9 34 56 AM](https://user-images.githubusercontent.com/46171897/120926362-80a57880-c6aa-11eb-87c6-1eb9f80efd4d.png)
